packages/Provider/src/intl/en-US.json

```diff
@@ -0,0 +1,3 @@
+{
+    "testMessage": "test message"
```

Comment:
Hmm i don't like the idea of having localized test messages. those folders are being shipped to npm, and if we were to add non-test strings, then both production and tests strings would be loaded by client applications